### ISSUE TITLE
добавить настоящие тесты

### ISSUE THREAD

**Issue by fedya** [rosa-abf/rosa-build#7]
То что сейчас есть это не совсем тест пакета.
Нужно устанавливать и прогонять бинарники через линтер какой, уверен выловим кучу сломанных симлинков и отсутствующих либ.

**Comment by saber716rus** [comment]
> Я давно хочу запускать пакеты в Xfvb, снимать скриншот и сравнивать с эталонным.
> Описание схемы:
> https://lists.altlinux.org/pipermail/sisyphus/2018-April/366621.html

Лучше тестинг, на подобии openqa. Данное решение успешно себя показало уже на 2х дистрах (suse и Федора)

**Comment by mikhailnov** [comment]
> Лучше тестинг, на подобии openqa. Данное решение успешно себя показало уже на 2х дистрах (suse и Федора)

OpenQA нечто подобное и делает. Тут нужно смотреть, оправдана ли ее сложность.

**Comment by saber716rus** [comment]
Так, как идет тенденция к большей автоматизации, то скорее она будет оправдана, чем нет.
